### ISSUE TITLE
Exception when running StatelessExamples - 6 - SelectExampleColumnAdjustment

### ISSUE THREAD

**Issue by larsw** [Microsoft/TrillSamples#6]
Exception / stack trace:
```
Unhandled Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentException: Argument types do not match                                                                                                                                          
   at System.Linq.Expressions.Expression.Bind(MemberInfo member, Expression expression)                                                                         
   at Microsoft.StreamProcessing.Streamable.<>c__DisplayClass198_0`2.<CreateAdjustColumnsExpression>b__7(KeyValuePair`2 o) in D:\a\1\s\Sources\Core\Microsoft.StreamProcessing\StreamableAPI\SelectExtensions.cs:line 222                                                                                                       
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()                                                                                              
   at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)                                                                              
   at System.Collections.Generic.SparseArrayBuilder`1.AddRange(IEnumerable`1 items)                                                                             
   at System.Collections.Generic.SparseArrayBuilder`1.ReserveOrAdd(IEnumerable`1 items)                                                                         
   at System.Linq.Enumerable.Concat2Iterator`1.ToArray()                                                                                                        
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)                                                                                             
   at Microsoft.StreamProcessing.Streamable.CreateAdjustColumnsExpression[TOld,TNew](IDictionary`2 newColumnFormulas, NewExpression newExpression) in D:\a\1\s\Sources\Core\Microsoft.StreamProcessing\StreamableAPI\SelectExtensions.cs:line 221                                                                               
   at Microsoft.StreamProcessing.Streamable.Select[TKey,TOld,TNew](IStreamable`2 source, Expression`1 initializer, IDictionary`2 newColumnFormulas) in D:\a\1\s\Sources\Core\Microsoft.StreamProcessing\StreamableAPI\SelectExtensions.cs:line 114                                                                              
   at StatelessExamples.Program.SelectExampleColumnAdjustment() in C:\TrillSamples\TrillSamples\StatelessExamples\Program.cs:line 180                           
   --- End of inner exception stack trace ---                                                                                                                   
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)                                            
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)                                             
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)                                                                                      
   at StatelessExamples.Program.Main(String[] args) in C:\TrillSamples\TrillSamples\StatelessExamples\Program.cs:line 271                                       
```

**Comment by peterfreiling** [comment]
Thanks. This has been fixed with [PR #18 in the Trill Repo](https://github.com/Microsoft/Trill/pull/18). We will work to publish a new Trill nuget package and update the Sample versions.

**Comment by peterfreiling** [comment]
@larsw this is now fixed with PR #8, which updated the Trill Nuget version.
